wtr-utils.js

```diff
@@ -9,6 +9,14 @@ const group = process.argv.indexOf('--group') !== -1;
 
 const NO_UNIT_TESTS = ['vaadin-icons', 'vaadin-lumo-styles', 'vaadin-material-styles'];
 
+/**
+ * Check if lockfile has changed.
+ */
+const isLockfileChanged = () => {
+  const log = execSync('git diff --name-only origin/master HEAD').toString();
```

Comment:
Is this working only in a branch? Though, I see that we don't run tests at all in the master branch, except Sonar.

Reply:
Good point, this needs to be `origin/20.0` in this case. I will fix this. Otherwise you are correct.